test/SlowTests/Issues/RavenDB_23132.cs

```diff
@@ -0,0 +1,140 @@
+﻿using System;
+using System.Linq;
+using System.Threading.Tasks;
+using FastTests;
+using Raven.Client.Documents.Operations.Indexes;
+using Raven.Client.Documents.Smuggler;
+using Tests.Infrastructure;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace SlowTests.Issues
+{
+    public class RavenDB_23132 : RavenTestBase
+    {
+        private readonly DatabaseItemType _operateOnTypes = DatabaseItemType.Documents |
+                                                            DatabaseItemType.Indexes |
+                                                            DatabaseItemType.TimeSeries;
+
+        public RavenDB_23132(ITestOutputHelper output) : base(output)
+        {
+        }
+
+        [RavenFact(RavenTestCategory.Indexes | RavenTestCategory.TimeSeries)]
+        public async Task DeleteTimeSeriesShouldNotThrowNotSupportedException_mapReduceIndexWithOutputToCollection()
+        {
+            using (var store = GetDocumentStore())
+            {
+                await store.Maintenance.SendAsync(new CreateSampleDataOperation(operateOnTypes: _operateOnTypes));
+
+                var indexName = "Companies/StockPrices/TradeVolumeByMonth";
+
+                await WaitAndAssertForValueAsync(async () =>
+                {
+                    var indexNames = await store.Maintenance.SendAsync(new GetIndexNamesOperation(start: 0, pageSize: int.MaxValue));
+                    return indexNames.Contains(indexName, StringComparer.OrdinalIgnoreCase);
+                }, true);
+
+                // add OutputReduceToCollection
+                var index = await store.Maintenance.SendAsync(new GetIndexOperation(indexName));
+                index.OutputReduceToCollection = "StockPricesTradeVolumeByMonth";
+                await store.Maintenance.SendAsync(new PutIndexesOperation(index));
+
+                try
+                {
+                    Indexes.WaitForIndexing(store);
+                }
+                catch
```

Comment:
Why do we have try / catch here and in other places?

Reply:
the try/catch block is there to ignore potential errors from `Indexes.WaitForIndexing(store);`. this allows irrelevant errors to be thrown while focusing specifically on the relevant index error. 
 I also added a comment to make this clearer.

Reply:
We have `bool allowErrors = false` for that purpose, don't we?

Reply:
yes, changed